include/config/config_graphics.h

```diff
@@ -138,12 +138,13 @@
 /**
  * Limits the horizontal fov on emulator like on console. May break viewport widescreen hacks.
  */
-// #define HORIZONTAL_CULLING_ON_EMULATOR
+// #define CULLING_ON_EMULATOR
```

Comment:
Is this still horizontal fov or does it affect vertical as well? If so, please update comment.